### PR TITLE
Pre commit hook

### LINKED ISSUE #42: Block commits on the main branch
Commits directly on the main branch shouldn't be allowed as the main branch is working as a 'production' branch right now, as well as a 'dev' branch that needs review.

### PR DESCRIPTION
**Pull request by glaubernespoli**
Creating a pre-commit script that runs on the pre-commit hook. The script blocks commits on the main branch.

### CODE REVIEW

File: hooks/pre-commit [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+branch="$(git rev-parse --abbrev-ref HEAD)"
+
+if [ "$branch" = "main" ]; then
+  echo "You can't commit directly to main branch."
```
Comment (Manonmani-s) [inline hooks/pre-commit:6]: I think the message should be more clear.

File: package.json [hunk 2]
```diff
@@ -76,7 +76,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged",
+      "pre-commit": "sh hooks/pre-commit && lint-staged",
```
Comment (Manonmani-s) [inline package.json:79]: lint-staged should come before the pre-commit hook.

**Review by Manonmani-s — CHANGES_REQUESTED** [review]
Changes needed

File: hooks/pre-commit (hunk 1 above)
Comment (glaubernespoli) [reply hooks/pre-commit:6]: I think the message is clear enough.

File: package.json (hunk 2 above)
Comment (glaubernespoli) [reply package.json:79]: Actually the pre-commit hook should come first, so the lint-staged doesn't even run in a place where it shouldn't.

**Review by Manonmani-s — APPROVED** [review]
Everything looks fine

**Pull request merged**